README.md

```diff
@@ -5,7 +5,7 @@
 
 ## About
 
-GitHub Action to extract metadata (tags, labels) for Docker. This action is particularly useful if used with
+GitHub Action to extract metadata such as tags from Git reference and GitHub events to tag and label Docker images. This action is particularly useful if used with
```

Comment:
Would prefer something like:

```
GitHub Action to extract metadata from Git reference and GitHub events. This action is particularly useful if used with Docker Build Push action to tag and label Docker images.
```

Also sign-off is missing for DCO.

Reply:
Ok, I'll make those changes!